src/main.js

```diff
@@ -27,6 +27,7 @@ import { faCircleQuestion } from "@fortawesome/free-solid-svg-icons";
 import { faBookmark } from "@fortawesome/free-solid-svg-icons";
 import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { faTableList } from "@fortawesome/free-solid-svg-icons";
+import makeAPI from "@/services/api";
```

Comment:
why are we going to the trouble to create an axios instance and pass it around? if it is just to set the `withCredentials` option then I'd rather just set that at execution time specifically when we are using cookies.  there are many other places we use axios with 3rd party services that won't be using `withCredentials`

Reply:
@dockabernathy01 you didn't remove this